.github/workflows/snapshot_pytorch_android.yml

```diff
@@ -0,0 +1,36 @@
+name: Snapshot PyToech Android
```

Comment:
To match our current naming convention, should this name (and file name) be "Native Snapshot PyTorch Android" and native_snapshot_pytorch_android.yml?

Reply:
this is for ai.djl.android package not for pytorch native(libtorch) or pytorch jni